db-service/test/cqn4sql/model/booksWithExpr.cds

```diff
@@ -0,0 +1,71 @@
+namespace booksCalc;
+
+
+
+entity Books {
+  key ID : Integer;
+  title : String;
+  author : Association to Authors;
+
+  a : Integer;
+  b : Integer;
+  length : Decimal;
+  width : Decimal;
+  height : Decimal;
+  stock : Integer;
+  price : Decimal;
+
+
+  // ---
+  c1 = a + b;
+  c2 : Integer = a + b;
+  c3 = a - b;
+
+  ctitle = substring(title, 3, stock);
+
+  // -- nested
+  cc1 = c1 * c3;
+  cc2 = cc1 / c1;
+  area : Decimal = length * width;
+  volume : Decimal = area * height;
+  storageVolume : Decimal = stock * volume;
+
+  // -- with paths
+  authorLastName = author.lastName;
+  authorName = author.name;
+  authorAdrText = author.addressText;
+
+  ca3 = f.ca; // F:ca is a calculated element with an assoc path
```

Comment:
i can't find `f` anywhere, is this a typo or how does this work?

Reply:
I started using "simple" names like f, c, ... and then changed to using "nice" names like author, name, ...
In the state I put into the PR there still is a mixture of both. Need to clean that up.